src/main/java/com/dev/cinema/service/AuthenticationServiceImpl.java

```diff
@@ -1,28 +1,39 @@
 package com.dev.cinema.service;
 
+import com.dev.cinema.model.Role;
+import com.dev.cinema.model.RoleType;
 import com.dev.cinema.model.User;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
     private final UserService userService;
     private final ShoppingCartService shoppingCartService;
+    private final RoleService roleService;
 
     @Autowired
     public AuthenticationServiceImpl(UserService userService,
-                                     ShoppingCartService shoppingCartService) {
+                                     ShoppingCartService shoppingCartService,
+                                     RoleService roleService) {
         this.userService = userService;
         this.shoppingCartService = shoppingCartService;
+        this.roleService = roleService;
     }
 
     @Override
     public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
+        user.setRoles(getUserRoles());
         User userFromDB = userService.add(user);
         shoppingCartService.registerNewShoppingCart(userFromDB);
         return userFromDB;
     }
+
+    private List<Role> getUserRoles() {
+        return List.of(roleService.getRoleByName(RoleType.USER.toString()));
+    }
```

Comment:
Why is this method in AuthenticationService? Also, is ADMIN prohibited to be an USER?